lib/QubitXmlImport.class.php

```diff
@@ -819,7 +819,24 @@ public static function replaceLineBreaks($node)
       }
       else
       {
-        $nodeValue .= preg_replace('/[\n\r\s]+/', ' ', $child->nodeValue);
+        if($child->tagName=='extent') 
+        {
+          foreach ($child->childNodes as $childChild)
+          {
+            if ($childChild->nodeName == 'lb')
+            {
+              $nodeValue .= "\n";
+            } 
+            else
+            {
+              $nodeValue .= preg_replace('/[\n\r\s]+/', ' ', $childChild->nodeValue);
+            }
+          }
+        } else {
+
+          $nodeValue .= preg_replace('/[\n\r\s]+/', ' ', $child->nodeValue);
+
+        }
```

Comment:
The else and the bracket should be both in new lines, like the other else before. And please remove the empty lines in the block if there is only one line, so it should be ...

`}
else
{
  $nodeValue .= preg_replace('/[\n\r\s]+/', ' ', $child->nodeValue);
}`